File: examples/javascript/constructs/array_library.js
// Ported from examples/lumen/constructs/array_library.lm by scripts/port_examples.py; edit the Lumen original, not this file.
const a = [1, 2, 3];
const b = [4, 5];
const both = array_concat(a, b);
console.log(both);
console.log(array_slice(both, 1, 4));
console.log(array_index_of(b, 5));
console.log(array_index_of(b, 9));
console.log(array_contains(a, 2));
console.log(array_contains(a, 7));
console.log(array_reverse(both));
console.log(array_reverse([]).length);

File: examples/python/constructs/array_library.py
# Ported from examples/lumen/constructs/array_library.lm by scripts/port_examples.py; edit the Lumen original, not this file.
a = [1, 2, 3]
b = [4, 5]
both = array_concat(a= a, b= b)
print(both)
print(array_slice(a= both, start= 1, stop= 4))
print(array_index_of(a= b, x= 5))
print(array_index_of(a= b, x= 9))
print(array_contains(a= a, x= 2))
print(array_contains(a= a, x= 7))
print(array_reverse(a= both))
print(len(array_reverse(a= [])))
